### ISSUE TITLE
Landing page animation doesn't work on IOS or Safari

### ISSUE THREAD

**Issue by zeim839** [ufosc/Club_Website_2#118]
The ThreeJS animation library is currently imported via an importmap statement in `views/index.js`. Importmap is still experimental, so its not supported by all browsers. This causes Safari, any iOS browser, and possibly Firefox to fail when starting the animation. 

**Solution:**
Add an import shim script before [line 13 of views/index.js](https://github.com/ufosc/Club_Website_2/blob/main/views/index.ejs#L13). The following script should work:
```html
<script async src="https://unpkg.com/es-module-shims@1.3.6/dist/es-module-shims.js"></script>
```

**Comment by malangadan** [comment]
I want to work on this.

**Comment by danielwildsmith** [comment]
Implemented in #124.
Thanks @malangadan (first PR!!)
